packages/ckeditor5-dev-tests/tests/utils/fileoptiontoglob.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* jshint mocha:true */
+
+'use strict';
+
+const { expect } = require( 'chai' );
+const sinon = require( 'sinon' );
+const path = require( 'path' );
+
+describe.only( 'dev-tests/utils', () => {
+	let fileOptionToGlob;
+
+	beforeEach( () => {
+		sinon.stub( path, 'join', ( ...chunks ) => chunks.join( '/' ) );
```

Comment:
I think that it's better to use sandbox. Lets do not overcomplicate our problem with tests :D